index.js

```diff
@@ -70,5 +76,52 @@ function toggleColor(element) {
 
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+
+
+function increaseLikes() {
+  const newLikes = parseInt(likes.textContent) + 1
+  likes.textContent = `${newLikes} Likes`
+}
+
+
+likeBtn.addEventListener("click", increaseLikes)
+
 
 /***** Deliverable 3 *****/
+
+// create variable for form, link and description
+const form = document.querySelector("#new-goal-form")
+
+
+
+// const goalId = player.goals.length + 1
+// haha i thought i was gonna be all clever with player.goals.length + 1 as the id -- but the input data doesn't persist. Thus all my new links would have a goal id of 3 lolol. back to the drawing board!
+
+const ul = document.querySelector("ul")
+const goalId = ul.children.length +1
+// boo this didn't work either. All of my new data-ids are 3.... why is that?
+  
+
+// will need event listener on form
+form.addEventListener("submit", function (e) {
+  e.preventDefault()
+
+  // will need newGoal with attributes
+// pass newGoal into renderGoal
+
+  const link = e.target["link"].value
+  const desc = e.target["description"].value
+
+  const newGoal = {
+    id: goalId,
+    link: link,
+    description: desc
+  }
+
+  renderGoal(newGoal)
+
+  e.target.reset()
+})
```

Comment:
Looks good! If you want to save on variables declarations, you could also create your object like so:
```javascript
  const newGoal = {
    id: player.goals[length - 1].id + 1,
    link: e.target["link"].value,
    description: e.target["description"].value
  }
```